Yosemite/Yosemite/Stores/AppSettingsStore.swift

```diff
@@ -749,6 +781,7 @@ enum AppSettingsStoreErrors: Error {
     case noProductsSettings
     case writeProductsSettings
     case noEligibilityErrorInfo
+    case noLastJetpackBenefitsBannerDismissedTime
```

Comment:
I think this case is not used anywhere - should it be removed?

Reply:
Yup this isn't needed anymore! removed in https://github.com/woocommerce/woocommerce-ios/pull/5505/commits/5d82aedc78b7120b7d8d1845750712195832e9d7